src/SOS/SOS.Hosting/RuntimeWrapper.cs

```diff
@@ -158,12 +158,10 @@ protected override void Destroy()
             }
             if (_dacHandle != IntPtr.Zero)
             {
-                DataTarget.PlatformFunctions.FreeLibrary(_dacHandle);
```

Comment:
The only thing is I'd add a comment here. It looks odd that this doesn't follow the pattern on the DBI.